packages/babel-types/test/converters.js

```diff
@@ -71,10 +71,10 @@ describe("converters", function () {
     });
     it("fail if class expression has no id", function () {
       const node = t.classExpression(null, null, t.classBody([]), []);
-      assert.throws(function() {
+      expect(function() {
         t.toStatement(node);
-      });
-      assert.strictEqual(t.toStatement(node, /* ignore = */ true), false);
+      }).toThrow();
```

Comment:
Would be nice to specify _what_ should be thrown (jest can take in a constructor (for instanceof check), string or regexp), but that definitely should be done in a later PR.